File: solutions/javascript/SingleElementinaSortedArray/SingleElementinaSortedArray.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNonDuplicate = function(nums) {
    for (let i = 0; i < nums.length; i++) {
        if (i % 2 === 0) {
            if ((i < nums.length - 1 && nums[i] != nums[i + 1]) || i === nums.length - 1) {
                return nums[i];
            }
        }
    }
};

File: solutions/java/SingleElementinaSortedArray/SingleElementinaSortedArray.java
class Solution {
    public int singleNonDuplicate(int[] nums) {
        for (int i = 0; i < nums.length; i++) {
            if ((i < nums.length - 1 && nums[i] != nums[i + 1]) || i == nums.length - 1) {
                return nums[i];
            }
            i++;
        }
        return -1;
    }
}